src/Bijectors.jl

```diff
@@ -253,12 +265,10 @@ include("utils.jl")
 include("interface.jl")
 include("chainrules.jl")
 
-Base.@deprecate forward(b::AbstractBijector, x) NamedTuple{(:rv,:logabsdetjac)}(with_logabsdet_jacobian(b, x))
+Base.@deprecate NamedBijector(bs) NamedTransform(bs)
 
-@noinline function Base.inv(b::AbstractBijector)
-    Base.depwarn("`Base.inv(b::AbstractBijector)` is deprecated, use `inverse(b)` instead.", :inv)
-    inverse(b)
-end
+Base.@deprecate Exp() elementwise(exp) false
+Base.@deprecate Log() elementwise(log) false
```

Comment:
Are deprecations needed if it is a breaking release? Or would it be sufficient to add them to some changelog/announcement/NEWS.md?

Reply:
Removed!